libs/blocks/aside/aside.css

```diff
@@ -370,19 +448,84 @@
   justify-content: flex-start;
 }
 
+.aside.split .foreground.container .text .icon-stack-area {
+  display: flex;
+  flex-flow: row wrap;
+  flex-direction: column;
+  gap: 12px;
+  margin: -8px 0 var(--spacing-s);
+  width: 100%;
+  padding: 0;
+  list-style-type: none;
+}
+
+.aside.split .foreground.container .text .icon-stack-area li,
+.aside.split .foreground.container .text .icon-stack-area li a {
+  display: flex;
+  align-items: center;
+  gap: 12px;
+  font-weight: bold;
+}
+
+.aside.split .foreground.container .text .icon-stack-area li picture {
+  flex-shrink: 0;
+}
+
+.aside.center:not(.notification):not(.split) .foreground.container {
+  padding: 0;
+}
+
+.aside.center:not(.notification):not(.split) .foreground.container .text {
+  margin: 80px 0;
+  text-align: center;
+  padding: 0;
+}
+
+.aside.center:not(.notification):not(.split) .foreground.container .icon-area {
+  max-width: 100%;
+}
+
+.aside.center:not(.notification):not(.split) .foreground.container .text .action-area {
+  justify-content: center;
+}
+
+.aside:not(.notification) .foreground.container .text .detail-m {
+  margin-bottom: var(--spacing-xs);
+}
+
+.aside.split .image.format {
+  display: flex;
+}
+
+.aside.split.bio .foreground.container .text .icon-area {
+  display: none;
+}
+
+
+.aside.center:not(.notification):not(.split) .foreground.container .text .icon-area {
+  height: 83px;
+}
+
+.aside.center:not(.notification):not(.split) .foreground.container .text .icon-area img {
+  height: 83px;
```

Comment:
Where is this value coming from?
```
 --icon-size-xxl: 80px;
  --icon-size-xl: 64px;
  --icon-size-l: 56px;
  --icon-size-m: 40px;
  --icon-size-s: 32px;
  --icon-size-xs: 24px;
  --icon-size-xxs: 16px;

Reply:
@ryanmparrish The height for center aside icon is 83px for desktop and mobile screen sizes as per the XD shared for the enhancement ticket.

Reply:
This number seems inconsistent w/ our size tokens. Is there room to discuss with design/XD author to keep our size expectations consistent?

Reply:
@ryanmparrish XD team has confirmed that the height should be 80px across screen sizes. I have make the corresponding changes in the PR.